YSE_App/galaxies/path.py

```diff
@@ -95,10 +95,14 @@ def ingest_path_results(itransient:FRBTransient,
             galaxy.redshift_err = icand.redshift_err
             galaxy.redshift_source = icand.redshift_source
             galaxy.redshift_quality = 1
-        elif hasattr(icand,'redshift_type') and icand.redshift_type == 'photo-z':
-            galaxy.photoz = icand.redshift
-            galaxy.photoz_err = icand.redshift_err
-            galaxy.photoz_source = icand.redshift_source
+
+        # Grab photo-zs from PS1
+        elif hasattr(icand, 'z_phot_median'):
+            galaxy.photoz = icand.z_phot_median
+            
+        # Grab photo-zs from DECaLS
+        elif hasattr(icand, 'z_phot_median'):
```

Comment:
isn't this identical to line 100 ?

Reply:
You're right! Need for separate PS1/DECaLS calls now deprecated. Have removed the duplicate lines.